parser/v2/parser.go

```diff
@@ -179,6 +183,18 @@ func (p *Parser) ParseEthLogs(_ context.Context, eventsData types.EventsData) (*
 	return &types.EventsParsedResult{EVMEvents: len(parsed), ParsedEvents: parsed}, nil
 }
 
+func (p *Parser) ParseMultisigEvents(ctx context.Context, txs []*types.Transaction, tipsetCid string, tipsetKey filTypes.TipSetKey) (*types.MultisigEvents, error) {
+	multisigTxs, err := p.helper.FilterTxsByActorType(ctx, txs, manifest.MultisigKey, tipsetKey)
+	if len(txs) > 0 {
```

Comment:
could return here if len == 0

Reply:
The if is just for debugging log, also, I need that if there are no txs go to the other method so that it returns empty arrays and then we process that in the state table